### PR TITLE
New Exchange Exmo

### PR DESCRIPTION
**Pull request by vitalisator**
Hi,
wants to give some back to this interesting software

- adds new Exchange https://exmo.com/en/api
- resolve the QuadrigaCX Bug, it was not possible to send a order bacause they accept only 8 digits after decimal point

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -53,6 +54,10 @@ struct Balance {
 // 'main' function.
 // Blackbird doesn't require any arguments for now.
 int main(int argc, char** argv) {
+  // Test Exchange functions
```
Comment (alfonsoperez) [inline src/main.cpp]: maybe we can remove this?

**Comment by butor** [comment]
Thanks @vitalisator!

**Comment by rubiwane** [comment]
@vitalisator great job, could you tell me where you modify the code for fixing QuadrigaCX bug?
I don't find it.

File: src/exchanges/quadrigacx.cpp [hunk 2]
```diff
@@ -70,27 +70,32 @@ std::string sendLongOrder(Parameters& params, std::string direction, double quan
...
-                  << std::setprecision(6) << quantity << " @ $"
+                  << std::setprecision(8) << quantity << "@$"
                   << std::setprecision(2) << price << "...\n";
-  std::string pricelimit = std::to_string(price);
-  std::string volume = std::to_string(quantity);
+  
+  std::ostringstream oss;
+  // Quadriga don't accept amount longer that 8 digits after decimal point
+  // Its a workaround, would be better to trim only digits after decimal point. 
+  oss << std::fixed << std::setprecision(8) << quantity;
```
Comment (vitalisator) [inline src/exchanges/quadrigacx.cpp]: @rubiwane , this is the quadriga fix for precision bug

Comment (rubiwane) [reply src/exchanges/quadrigacx.cpp]: Thanks you @vitalisator Just to be sure, is it included in the last update?

**Pull request merged**